### PR TITLE
Added params for libvirt_host

### PR DESCRIPTION
**Pull request by sghai**
This is required to cover step 1) in issue #3063

### CODE REVIEW

File: robottelo.properties.sample [hunk 1]
```diff
@@ -188,3 +188,12 @@ ssh_key=
 # Parameter for deciding whether conduct initial sync or resync
 # 'resync' denotes resync; 'sync' denotes initial sync
 # sync_type='sync'
+
+# [libvirt_host]
```
Comment (kbidarkar) [inline robottelo.properties.sample]: How about we making this [compute_resources] ?
I am not asking to change this, but rather so we discuss and think about it, if it is OK ?

Thinking In future all the other(vmware, rhevm, openstack) CR details also could go below this.

Comment (sghai) [inline robottelo.properties.sample]: +1
Updating now.. anyway, I'm using `libvirt_` keyword.

**Comment by sthirugn** [comment]
ACK

File: robottelo.properties.sample (hunk 1 above)
Comment (kbidarkar) [inline robottelo.properties.sample]: :+1:  for `libvirt_` keyword.

Comment (elyezer) [inline robottelo.properties.sample]: The main point here is, when we have more than one compute resource, like openstack and we just want to use libvirt, having all under compute_resources is good in terms of organization, but we may find problems with validation, like, in the example mentioned, don't knowing what should be required the libvirt_hostname and/or the openstack_hostname...

Comment (kbidarkar) [inline robottelo.properties.sample]: ok, if using separate section helps isolate things then that's good. Ok, I agree separating things would quickly help us know what is required and what is not , for a particular CR.

Also sometimes certain CR's have multiple variables too.

**Comment by elyezer** [comment]
ACK

**Pull request merged**